Sources/HeroContext.swift

```diff
@@ -134,7 +134,9 @@ extension HeroContext {
       } else if let barView = view as? UINavigationBar, barView.isTranslucent {
```

Comment:
Not sure how tvOS's navigation bar works. But i think we should not go in to this `else if` statement. The navigationBar snapshot used here is pretty dependent on iOS's navigation bar implementation. Lets limit this `else if` to be only available on iOS